minio/helpers.py

```diff
@@ -271,6 +274,17 @@ def get_target_url(endpoint_url, bucket_name=None, object_name=None,
 
     return ''.join(url_components)
 
+def is_valid_http_client(client):
+    """
+    Verify if endpoint is valid urllib3 http client.
+
+    :type client: urllib3.poolmanager.PoolManager
+    :return: True if the client is valid, Raise :exc:`InvalidArgumentError`
+       otherwise.
+    """
+    if isinstance(client, urllib3.poolmanager.PoolManager):
+        return True
+    raise InvalidArgumentError('HTTP client should be of instance `urllib3.poolmanager.PoolManager`')
```

Comment:
This is not pythonic.  you throw an exception on `isinstance()` failure and return nothing.  (There is no nothing in python, you get `None`).  IMO this as separate function is not needed.